pyproj/transformer.py

```diff
@@ -53,6 +53,9 @@ def from_proj(proj_from, proj_to, skip_equivalent=False):
         skip_equivalent: bool, optional
             If true, will skip the transformation operation if input and output 
             projections are equivalent. Default is false.
+        static_xy: bool, optional
+            If true, will force the x,y axis to always be in the same order.
+            Default is false.
```

Comment:
While writing the above, it might have become more clear :-) 
With  "x,y axis always in the same order", you mean "the *input* and *output* x, y will always be in the same order"? Meaning if the input is in east/north, the output will also be east/north?

Maybe we can just be more explicit? As now the "always same order" is a bit general, while in practice it means "always east, north order" (eg it will never be north, east order for both input/output).

In the PROJ docs, they also use the words "uniform axis order (and thus do not care about axis order mandated by the authority defining the CRS)". 
Also taken from there (slightly modified): If this keyword is toggled, the `transform` method will accept as input and return as output coordinates using the traditional GIS order, that is longitude, latitude (followed by elevation, time) for geographic CRS and easting, northing for most projected CRS.

Reply:
Thanks for the idea - I pretty much took your explanation almost word for word as it is very clear and makes a lot of sense. I left the argument as `static_xy` as I was not sure of a better alternative. I thought of maybe `gis_xy_order` or something, but I currently like `static_xy` better.

Reply:
What about `always_xy` instead of `static_xy` ? 
(that's how you explain it in the transformer.rst: "If you prefer to keep your axis order as always x,y, you can use the ...")

Reply:
I like it 👍